feature-detects/indexeddb2.js

```diff
@@ -0,0 +1,22 @@
+/*!
+{
+  "name": "IndexedDB 2.0",
+  "property": "indexeddb2",
+  "tags": ["storage"],
+  "caniuse": "indexeddb2",
```

Comment:
please add this also to the test/browser/integration/caniuse.js file

Reply:
It's already in the PR: https://github.com/Xenonym/Modernizr/blob/feature/add-indexeddb-2/test/browser/integration/caniuse.js#L99